dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1611,6 +1611,16 @@ std::tuple<winrt::Rect, winrt::Thickness, winrt::Thickness> TeachingTip::Determi
     if (!m_useTestWindowBounds)
     {
         windowBounds = winrt::Window::Current().CoreWindow().Bounds();
```

Comment:
Recommend using inline lambda here to avoid calling CoreWindow.Bounds when we don't need it.
```c++
auto windowBounds = []() {
    if (m_useTestWindowBounds)
    {
        return m_testWindowBounds;
    }
#ifdef USE_INSIDER_SDK
    if (winrt::IUIElement10 uiElement10 = *this)
    {
        if (auto xamlRoot = uiElement10.XamlRoot())
        {
            return winrt::Rect { 0, 0, xamlRoot.Size().Width, xamlRoot.Size().Height };
        }
    }
#endif // USE_INSIDER_SDK

    return winrt::Window::Current().CoreWindow().Bounds();
}();
```